### PR TITLE
pages/*: add standard translation and links

### PR DESCRIPTION
**Pull request by kbdharun**
Signed-off-by: K.B.Dharun Krishna <kbdharunkrishna@gmail.com>

## Changes

- Add Finish translation to More information link.
- Standardize period in More information link for Bengali, Hindi, and Nepali from `.` to `।`.
- Run `set-alias-page.py` script (as there has been 3 language additions since the last time I ran this script last year) to automatically generate alias pages based on the translation template.
- Run `set-more-info-link.py` script to automatically update links throughout the translation pages.

----

**Note**: ~This PR won't be ready for review for at least the next couple of days~, while I work on eliminating the false positives (which I am aware of), especially with the Windows pages (we had similar ones in #9672 too, and I manually went through all the pages and eliminated them).

----

**FAQ**

**How do I check/review these changes?**

Since this PR modified a lot of lines (in hundreds of pages), GitHub diff wouldn't load even a fraction of the changed files and would often crash. So once it is ready for review you can either create a GitHub codespace or use VSCode locally with the GitHub (PR and main extensions loaded). This allows seamless reviewing and keeping track of all the changes).

### CODE REVIEW

File: pages.de/common/just.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# just
+
+> Dieser Befehl ist ein Alias von `just.1`.
```
Comment (gutjuri) [inline pages.de/common/just.md:3]: `just` is an alias for two pages. See the english version: https://github.com/tldr-pages/tldr/blob/main/pages/common/just.md

**Review by gutjuri — COMMENTED** [review]
Sometimes the alias pages are broken. When the  english version links to more than one page, the (german) translation only links to one page.

**Comment by kbdharun** [comment]
> Sometimes the alias pages are broken. When the english version links to mroe than one page, the (german) translation only links to one page.

Yep, this is exactly what I asked in the [chatroom about](https://matrix.to/#/!zXiOpjSkFTvtMpsenJ:gitter.im/$48bCLEXTnU64GPUIslDhZm7cSK-it4tdrURylv8Xf3g?via=gitter.im&via=matrix.org&via=one.ems.host).

I will drop off the whole message here:

> Hi, so I need your opinion on this PR https://github.com/tldr-pages/tldr/pull/11331. I have fixed the false positives for modified files. But there might be places in the new additions where a few commands might be missing (as the script only works translation pages matching the exact translation page template + 1 line more info link) and not any other modifications.
> Last time when we executed the script last year, we decided to leave the pages to be fully completed by translators and merged the PR. Should we do the same this time to reduce tedious reviewing process for maintainers or should we go with the process of manually completing the pages (for which we have native reviewers).
> I personally prefer the first option as working on independent PRs to complete the missing commands (on alias pages) would be better for reviewing.

**Comment by kbdharun** [comment]
> Maybe add `bn` to [this line](https://github.com/tldr-pages/tldr/blob/a00e12ca3901632b7d4ada5aaa6f09625819c26e/scripts/set-more-info-link.py#L89)?

I think I already did it in this PR. Will check the file and get back.

**Pull request merged**